v2/spiffex509/svid.go

```diff
@@ -0,0 +1,15 @@
+package spiffex509
+
+import (
+	"crypto"
+	"crypto/x509"
+
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+)
+
+// SVID represents a SPIFFE X509-SVID
+type SVID struct {
+	ID           spiffeid.ID
```

Comment:
can you add some comments to attributes? we know what is it but it can be usefull for new guys just learning about what an SVID is

Reply:
Done